torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -784,44 +785,79 @@ SerializationStorageContext& ScriptModuleSerializer::storage_context() {
   return storage_context_;
 }
 
+void save_mobile_module_to(
+    const Module& module,
+    const ExtraFilesMap& extra_files,
+    bool save_mobile_debug_info,
+    const std::function<size_t(const void*, size_t)>& writer_func) {
+  CompilationOptions options = getOptionsFromGlobal();
+  mobile::Module mod = jitModuleToMobile(module, options);
+  auto buffer = save_mobile_module_to_bytes(mod, extra_files);
+  writer_func(reinterpret_cast<char*>(buffer.data()), buffer.size());
```

Comment:
Consider casting to `void*` to directly match the signature of `writer_func`; same for another cast later in this file. Not a big deal though.